src/components/icon-tile-logo/docs.mdx

```diff
@@ -0,0 +1,52 @@
+---
+componentName: 'IconTileLogo'
+---
+
+Use the IconTileLogo component when representing the product in our interfaces.
+
+Renders a product icon in a brand-colored tile. Note that this IconTileLogo
+is currently styled to match dev-portal designs, and differs in size and
+background coloration from [the IconTileLogo documented in the
+HashiCorp Design System](https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/HDS-Product---Components-%5BWIP%5D?node-id=1377%3A11992).
+
+<LiveComponent>{`
+    <IconTileLogo product="waypoint" />
+`}</LiveComponent>
+
+## Examples
```

Comment:
Have added what I think is an exhaustive set of Examples here. Happy to add more, or add any other info here, if it feels like anything is missing 👍